patch_recurrence/steps.py

```diff
@@ -39,10 +40,53 @@ def generate_patches(scaled_imgs, constants):
         patches.append(img_patches)
     return patches
 
+#..............................................................................................#
+# purva's functions
+
+# exactly like generate_patches without the std deviation condition
+def generate_all_patches(scaled_imgs, constants):
```

Comment:
Is it possible for you to integrate this code into the existing `generate_patches` function? We are repeating code unnecessarily here. Perhaps, you could build a separate list of all patches